CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 0.8.2 (UNRELEASED)
+--------------------------
+
+- Administrators:
+    - Adds configuration environment variable, ``reana_server.environment.REANA_SCHEDULER_REQUEUE_COUNT``, to set requeue count for submitted workflows in case of scheduling error.
```

Comment:
Minor: we should also mention that cluster could be simply busy for the workflows to be requeued. Similarly as you mentioned [here](https://github.com/reanahub/reana-server/blob/5c046f7dde1d0881a3e0a99aa82881657aa0945b/reana_server/config.py)